### ISSUE TITLE
Why path of building working super slow ?

### ISSUE THREAD

**Issue by lilr5** [Openarl/PathOfBuilding#825]
my path of building working super slow 
I've tried unistall and re install path of building
I tried to reset my computer
my hardware:
intel pentium g4600 
asrock h110
adata 8gb ddr4 2133mhz ram
600w 80+ power supply
I havent grapich card im using my cpu s grapich card

**Comment by Dawndove** [comment]
Yeah i am having the same problem with lag. unable to use the program in current state.
AMD FX 4300 Quad core Processor
3.8 ghz
8 GB Ram
windows 10

**Comment by theenngee** [comment]
Have the same Problem:
Ryzen 1600
16GB RAM
Geforce 1060GTX 6GB
Windows 10 Install
Problem also persisted on my old System with an i5.
